src/main/java/org/ilastik/ilastik4ij/ui/IlastikPixelClassificationCommand.java

```diff
@@ -42,6 +43,21 @@
     @Parameter(label = "Output type", choices = {UiConstants.PIXEL_PREDICTION_TYPE_PROBABILITIES, UiConstants.PIXEL_PREDICTION_TYPE_SEGMENTATION}, style = "radioButtonHorizontal")
     public String pixelClassificationType;
 
+    @Parameter(label = "Use Mask?", persist=false, initializer = "initUseMask")
+    public boolean useMask=false;
+
+    protected void initUseMask(){
+        useMask = false;
+        //resolveInput("useMask"); //this makes the input not be rendered -.-
+    }
+
+    @Parameter(
+        label = "Prediction Mask",
+        required = false,
+        description = "An image with same dimensions as Raw Data, where the black pixels will be masked out of the predictions"
+    )
+    public Dataset predictionMask;
```

Comment:
Did you try:
```suggestion
    public Dataset predictionMask = null;
```
to see whether that solves the issue with a default value?